pylint/extensions/mccabe.py

```diff
@@ -134,23 +137,32 @@ def _subgraph_parse(
         self,
         node: _SubGraphNodes,
         pathnode: _SubGraphNodes,
-        extra_blocks: Sequence[nodes.ExceptHandler],
+        extra_blocks: Sequence[nodes.ExceptHandler | nodes.MatchCase],
     ) -> None:
-        """Parse the body and any `else` block of `if` and `for` statements."""
+        """Parse the body and any `else` block of `if`, `for` or `match` statements."""
```

Comment:
optional wording fix: "else" block only pertains to "if" and "for" nodes, not "match".

Reply:
I'm not sure what to do here, do you mind suggesting ?